cylc/flow/scripts/reinstall.py

```diff
@@ -295,7 +299,14 @@ def reinstall(
     # run post_install plugins
     if not dry_run:
         # don't run plugins in dry-mode
-        post_install(opts, src_dir, run_dir)
+        async with async_block():
```

Comment:
This ensures that any async tasks started by the plugins are awaited before we move on.

Rose fileinstallation will create async tasks but cannot await them (because it's called synchronously) so we have to do it here, otherwise `cylc vip` would start running the workflow before local file installation has finished.